msal-obo-sample/src/main/java/com/microsoft/azure/msalobosample/AADClaimsVerifier.java

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.azure.msalobosample;
+import org.springframework.security.oauth2.common.exceptions.InvalidTokenException;
+import org.springframework.security.oauth2.provider.token.store.JwtClaimsSetVerifier;
+import org.springframework.util.Assert;
+import org.springframework.util.CollectionUtils;
+import java.util.Arrays;
+import java.util.Map;
+
+public class AADClaimsVerifier implements JwtClaimsSetVerifier {
+    private static final String ISS_CLAIM = "iss";
+
+    private final String[] issuers;

```

Comment:
I would use set here, issuers should be unique

Reply:
As a class I would advise we follow the .net middleware team's Api surface ,where they allow both a single issuer and multiple issuers to be set. This is extremely useful for multi-tenant apps, who use the array to restrict the tokens presented to them from a select number of tenants. Note: My knowledge of Java is rudimentary, and I'm assuming this class will be used to verify tokens

Reply:
Changed to set. Added possibility for multiple accepted issuing tenants.